Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/templates/submenu.css

```diff
@@ -55,19 +84,28 @@ ul {
 	background-image: url("images/GEV_SubMenueImg5.png");
 }
 
+.bx-prev, .bx-next {
+  width: 30px;
+  position: relative;
+  height: 120px;
+  display: inline-block;
+}
 
-.catSubMenuItem {
-	float: left;
-	min-width: 100px;
-	padding: 100px 20px 30px;
-	text-align: center;
-	background-repeat: no-repeat;
-	background-position: center 30px;
+.bx-prev {
+  float: left;
 }
 
-.catSubMenuItem:hover {
-	background-color: #ccc;
+.bx-next {
+  float: right;
 }
 
-.catScrollRight {
+.disabled {
```

Comment:
Das ist ein zu allgemeiner Name, ich bin mir fast sicher, das der irgendwo anders in ILIAS schonmal verwendet wird. Wenn nicht kommt spätestens morgen ein Held auf diesen Namen =)